test/puppetca_http_api/puppetca_http_impl_test.rb

```diff
@@ -43,7 +43,30 @@ def search(key = 'foreman')
             'SHA512' => '83:BD:D2:32:30:F3:3E:69:7D:61:ED:A8:3F:3D:29:81:1C:96:AC:39:9B:A3:09:9E:61:9F:17:78:91:69:73:12:84:51:59:EE:93:42:AB:A8:34:72:41:43:B5:48:32:E7:3C:DE:85:13:5E:78:A5:C9:FD:A3:FF:54:53:7C:E6:03',
             'default' => 'F8:DA:15:EA:BD:2F:2D:D3:05:71:73:55:96:74:A4:97:2B:04:06:47:A8:8E:D2:C4:AB:8F:EC:3B:7C:0F:0A:EE'
           },
-          'not_after' => '2019-08-25T19:25:29UTC',
+          'not_after' => '2039-08-25T19:25:29UTC',
```

Comment:
Updated this date to be further in the future so the tests don't start failing later in the year.